### PR TITLE
US_62_T_69 Add recipient on message header

### PR DESCRIPTION
**Pull request by Kjha7**

### CODE REVIEW

File: ui/src/Components/ChatHeader.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import ChatHeaderRecipient from './ChatHeaderRecipient';
 
 const ChatHeader = ({msg}) => {
-  return <div class="header-container">{msg.subject}</div>;
+  return (
+    <div class="header-container">
+      {msg.subject}
+      <div class="component-header">
```
Comment (ishansarangi) [inline ui/src/Components/ChatHeader.js]: is this class required? the component header was used in the thread component.

File: ui/src/Components/message.css [hunk 2]
```diff
@@ -125,17 +125,22 @@ body {
...
   flex-shrink: inherit;
 }
-.header-container:hover {
-  overflow: visible;
+.header-container-recipient {
+  background-color: #ffffff;
+  padding: 12px 0px 12px 2px;
+  font-size: 12px;
 }
+/* .header-container:hover {
```
Comment (ishansarangi) [inline ui/src/Components/message.css]: this was done to show the truncated titles.  Is it still visible?

Comment (Kjha7) [reply ui/src/Components/message.css]: Yes, it is still visible.

File: ui/src/Components/ChatHeader.js (hunk 1 above)
Comment (Kjha7) [reply ui/src/Components/ChatHeader.js]: I have removed component-header div

**Pull request merged**